docs/zh_CN/sqls/query_language_elements.md

```diff
@@ -341,17 +341,17 @@ FROM table_name
 ORDER BY column1, column2, ... ASC|DESC;
 ```
 
-## Case Expression
+## Case表达式
```

Comment:
Chinese and English must have a space in between. The same problems happen in elsewhere in this pr.

Reply:
fixed.